docs/design/features/net-core-applications.md

```diff
@@ -13,7 +13,7 @@ NETStandard version supported | `netstandard2.1`
 
 ## FAQ
 **Q: What is a .NET Core application?**  
-**A:** A .NET Core application is an application that can run on any .NET Core runtime: CoreCLR (current), Mono (future). It can run on one of many .NET core platforms (Windows, OSX, Linux).  It relies on the host provided by the given runtime.  It's a composable framework built from the packages on which the application depends.  Its assembly loading policy permits newer versions of dependencies without any application configuration (e.g.: BindingRedirects are not required).
+**A:** A .NET Core application is an application that can run on any .NET runtime: CoreCLR (current), Mono (future). It can run on one of many .NET core platforms (Windows, OSX, Linux).  It relies on the host provided by the given runtime.  It's a composable framework built from the packages on which the application depends.  Its assembly loading policy permits newer versions of dependencies without any application configuration (e.g.: BindingRedirects are not required).
```

Comment:
I’d clarify something about the .net framework not being supported

Reply:
I am deleting this whole file. It is not up to date, and it covers topics that are covered in other places with more details.